proposals/incubation/cert-manager.md

```diff
@@ -0,0 +1,138 @@
+
+One prominent example of a cert-manager user might be the CNCF themselves, using cert-manager [in the DevStats project][DevStatsCertManager]!
+
+In addition we might measure some production use through the many large companies who have seen fit to create addons for cert-manager to help it better integrate into their environments. That includes (at the time of writing):
+
+- at least [49 public repositories][ExternalDNSSolvers] implementing an external ACME DNS-01 solver
+- at least [8 public repositories][ExternalIssuers] implementing complete external issuers for certificates
+
+Finally, we track a few users in a [USERS.md file][USERSmd] on our repository, although that file doesn't fully represent the scale of cert-manager's prominence in clusters today.
+
+### Committer Numbers
+
+> Have a healthy number of committers
+
+cert-manager currently has 9 maintainers across 3 organisations. Community contributions from many different organizations are very common and gladly received.
+
+The [companies-table DevStats dashboard][CompaniesTable] is illustrative. We've seen:
+
+- 4 different companies which have made over a thousand contributions each,
+- 8 different companies which have made over a hundred contributions each,
+- and over a thousand different companies which have contributed something to the project.
+
+In 2021, cert-manager saw [102 new first-time-contributors][NewContributors2021] to the project. Drilling down, cert-manager v1.7 saw at least 7 different individual community contributors, and one of the major features in that release was [contributed by a community member][ReleaseNotes17].
+
+2022 has started strongly too; at the time of writing there have been [21 new contributors][NewContributors2022] already in this calendar year.
+
+In terms of code changes only and ignoring other types of contribution, we see an [average of 10 different individual committers][CommittersCertManager] per month in the 6 months before February 2022 for cert-manager alone, rising to an [average of 20 per month][CommittersAllRepos] across all repositories in the same time period.
+
+### Ongoing Commits
+
+> Demonstrate a substantial ongoing flow of commits and merged contributions
+
+All recent releases of cert-manager have included major features contributed by both the core maintainer team and community members, driven by a contiuous flow of commits every month.
+
+Looking at CNCF DevStats, cert-manager saw an average of [39 pull requests opened][AvgOpenedPRsCertManager] per month in the 6 months before February 2022 (excluding bots), with an average of [30 PRs merged per month][AvgMergedPRsCertManager] in the same time period.
+
+The above numbers only count the main cert-manager repo; as mentioned previously, there are several cert-manager subprojects being regularly worked on too, including the cert-manager website.
+
+Across all repos we see an average of [78 merged PRs][AvgMergedPRsAllRepos] per month in the same time period as above.
```

Comment:
```suggestion
The above numbers only count the main cert-manager repo; as mentioned previously, there are several cert-manager subprojects being regularly worked on too, including the cert-manager website. Across all repos we see an average of [78 merged PRs][AvgMergedPRsAllRepos] per month in the same time period as above.
```

These two points seem to be directly tied together, so I would suggest putting them in the same paragraph.